### PR TITLE
shayan/Floating Zoom in/out and home buttons

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

**Comment by morteza-binary** [comment]
@shayan-binary-2 please solve conflicts

File: src/components/Menu.jsx [hunk 1]
```diff
@@ -42,23 +44,40 @@ class Menu extends Component {
...
+                            >
+                                {newDropdown}
+                            </div>
+                        </div>
+                    </div>,
+                    portalNode,
+                );
+            } else if (isMobile && modalNode) {
+                newDialog = ReactDOM.createPortal(
+                    <div className={`cq-modal-dropdown ${className || ''} ${open ? 'stxMenuActive' : ''}`}>
```
Comment (morteza-binary) [inline src/components/Menu.jsx]: @shayan-binary-2 ditto

File: src/components/NavigationWidget.jsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { connect } from '../store/Connect';
+import CrosshairToggle from './CrosshairToggle.jsx';
 import '../../sass/components/_ciq-navigation-widget.scss';
 
-import { ZoominIcon, ZoomoutIcon, HomeIcon, ScaleIcon } from './Icons.jsx';
+import { ZoominIcon, ZoomoutIcon, ScaleIcon } from './Icons.jsx';
```
Comment (morteza-binary) [inline src/components/NavigationWidget.jsx]: @shayan-binary-2 why is `HomeIcon` removed?

File: src/components/Menu.jsx [hunk 3]
```diff
@@ -42,23 +44,40 @@ class Menu extends Component {
...
-                                    {newDropdown}
-                                </div>
-                            </div>,
-                            modalNode,
-                        )
-                    || newDropdown;
+            let newDialog = newDropdown;
+            if (portalNode) {
+                newDialog = ReactDOM.createPortal(
+                    <div className={`smartcharts-${theme}`}>
```
Comment (morteza-binary) [inline src/components/Menu.jsx]: @shayan-binary-2 I'd be better to assign it to a variable and use it as the function parameter. It's more readable.

File: src/components/NavigationWidget.jsx (hunk 2 above)
Comment (shayan-binary-2) [reply src/components/NavigationWidget.jsx]: @morteza-binary HomeIcon used to be on the right hand, after that Ui/UX decide to move it to left on the top of navigation, and now we move back to the left side

**Pull request merged**